README.md

```diff
@@ -9,8 +10,8 @@ Ensure you have pip, python and virtualenv installed
 
 #### 1. Clone the repository.
 ```
-git clone https://github.com/stytchauth/stytch-b2b-flask-magic-links.git
-cd stytch-b2b-flask-magic-links
+git clone https://github.com/stytchauth/stytch-b2b-flask-quickstart.git
+cd stytch-b2b-flask-magic-quickstart
```

Comment:
Should this be `stytch-b2b-python-quickstart` instead?

Reply:
Hmm good question! Now that we've renamed our BE quickstarts to be the language instead of the framework -- probably! I guess there is a possibility that we might want to create a Django example app in the future, and then we'd need to rename again -- but we can deal with that if/when it happens!

Reply:
updated!

Reply:
Oops sorry another nit - we should remove the `magic` too, I think that was leftover from `magic-links`. I am also ok with it being `stytch-b2b-flask-quickstart`! I assume Flask is a commonly-known Python library.